components/quizSingle/ResultView.tsx

```diff
@@ -0,0 +1,70 @@
+import Link from "next/link";
+import { QuizRecord } from "~/models/ChinguQuiz/QuizRecord";
+import { Heading4, TextBodyBold, TextBodySmallBold } from "../shared/styles";
+import {
+  ResultPageContainer,
+  ResultTitleContainer,
+  ResultTileContainer,
+  ResultTile,
+  CodeBlock,
+  TileNumber,
+  SubmitQuizBtnStyled,
+} from "./styles";
+import ScoreGraph from "./ScoreGraph";
+
+export default function ResultView({
+  quizTitle,
+  quizRecord,
+}: {
+  quizTitle: string;
+  quizRecord: QuizRecord[];
+}) {
+  let percentage = Math.round(
+    (quizRecord.filter(r => r.correct).length * 100) / quizRecord.length
+  );
+  let totalQs = quizRecord.length;
+
+  return (
+    <ResultPageContainer>
+      <ScoreGraph percentage={percentage} />
+
+      <div>
+        <ResultTitleContainer>
+          <Heading4>{quizTitle}</Heading4>
+        </ResultTitleContainer>
+
+        <ResultTileContainer>
+          {quizRecord.map((record, i) => (
+            <ResultTile
+              correct={record.correct}
+              animationDelay={`${100 + i * 50}ms`}
+              key={`${quizTitle}-${i}`}
+            >
+              <div>
+                <TextBodyBold>
+                  Question:{" "}
```

Comment:
Is there a reason for an empty space here? .. It seems like the text and quiz number is aligned on either side anyway

Reply:
Yes, the formatter puts it in every time I change it and save 😐.